main/utils/textEntry.c

```diff
@@ -9,8 +9,9 @@
  *
  * @param endH Where the keyboard ends
  * @param elapsedUs How many ms have elapsed since last time function was called
+ * @return int16_t Top of textbox for nex element
```

Comment:
```suggestion
 * @return int16_t Top of textbox for next element
```

I use https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker